src/test/java/org/gridsuite/shortcircuit/server/service/ShortCircuitServiceTest.java

```diff
@@ -115,7 +103,8 @@ void testLogsMappersIsCalled() throws Exception {
         final UUID resultUuid = UUID.fromString("33333333-3333-3333-3333-333333333333");
         final String reporterId = "44444444-4444-4444-4444-444444444444";
         final ShortCircuitRunContext runContext = new ShortCircuitRunContext(networkUuid, null, null,
-                new ShortCircuitParameters(), reportUuid, reporterId, "AllBusesShortCircuitAnalysis", null, null);
+                new ShortCircuitParameters(), reportUuid, reporterId, "AllBusesShortCircuitAnalysis", null,
+                ShortCircuitAnalysis.find().getName(), null);
```

Comment:
Could you set null for provider instead of ShortCircuitAnalysis.find().getName()

Reply:
done